lib/Service/ColumnService.php

```diff
@@ -140,6 +150,8 @@ public function create(
 		$item->setNumberMin($numberMin);
 		$item->setNumberMax($numberMax);
 		$item->setNumberDecimals($numberDecimals);
+		$item->setNumberPrefix($numberPrefix !== null ?: '');
+		$item->setNumberSuffix($numberSuffix !== null ?: '');
```

Comment:
Couldn't the parameter then just be not-nullable and have an empty string as the default?